src/app/core/configurable-enum/enum-dropdown/enum-dropdown.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, Input, OnChanges, SimpleChanges } from "@angular/core";
+import { MatSelectModule } from "@angular/material/select";
+import { FormControl, ReactiveFormsModule } from "@angular/forms";
+import { ConfigurableEnumDirective } from "../configurable-enum-directive/configurable-enum.directive";
+import { NgForOf, NgIf } from "@angular/common";
+import {
+  CONFIGURABLE_ENUM_CONFIG_PREFIX,
+  ConfigurableEnumConfig,
+  ConfigurableEnumValue,
+} from "../configurable-enum.interface";
+import { BasicAutocompleteComponent } from "../basic-autocomplete/basic-autocomplete.component";
+import { ConfigService } from "../../config/config.service";
+
+@Component({
+  selector: "app-enum-dropdown",
```

Comment:
Is there a reason for making this a extra component (and not just move the code to the `EditConfigurableEnumComponent`